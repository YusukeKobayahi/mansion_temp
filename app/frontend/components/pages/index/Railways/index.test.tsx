import { render } from "@testing-library/react";
import RailWays from "~/components/pages/index/Railways";
import ToggleContextProvider from "~/components/Context/ToggleContext";
import SearchConditionContextProvider from "~/components/Context/SearchConditionContext";

describe("components/pages/index/Railways", () => {
  it("正常に表示される", () => {
    const { asFragment } = render(
      <SearchConditionContextProvider searchCondition={{}}>
        <ToggleContextProvider>
          <RailWays
            setTab={() => {
              return;
            }}
          />
        </ToggleContextProvider>
      </SearchConditionContextProvider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
