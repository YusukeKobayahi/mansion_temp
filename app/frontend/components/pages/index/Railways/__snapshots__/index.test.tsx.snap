// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/pages/index/Railways 正常に表示される 1`] = `
<DocumentFragment>
  <section
    class="container"
  >
    <p
      class="title"
    >
      駅・路線から探す
    </p>
    <div
      class="main"
    >
      <button
        class="trigger"
      >
        駅・沿線から探す
      </button>
    </div>
  </section>
</DocumentFragment>
`;
